src/components/search-package-list-item/search-package-list-item.test.js

```diff
@@ -0,0 +1,117 @@
+import { screen, render, fireEvent } from '@testing-library/react';
+
+import { Router } from 'react-router';
+import { createMemoryHistory } from 'history';
+
+import SearchPackageListItem from './search-package-list-item';
+
+jest.mock('../tags-label', () => () => (<div>TagsLabel component</div>));
+jest.mock('../hidden-label', () => () => (<div>HiddenLabel component</div>));
+
+const routerHistory = createMemoryHistory();
+
+const testTags = ['first', 'second', 'third'];
+
+const testEmptyProps = {};
+
+const testBasicProps = {
+  item: {
+    name: 'test-package-name',
+    providerName: 'test-provider-name',
+    isSelected: true,
+    titleCount: 1,
+    visibilityData: {
+      isHidden: false,
+    },
+    tags: {
+      tagList: testTags,
+    },
+  },
+  link: 'test-link-url',
+};
+
+const testFullProps = {
+  ...testBasicProps,
+  item: {
+    ...testBasicProps.item,
+    visibilityData: {
+      isHidden: true,
+    }
+  },
+  showProviderName: true,
+  showTitleCount: true,
+  showTags: true,
+};
+
+describe('Given SearchPackageListItem', () => {
+  const handleClick = jest.fn();
+
+  const renderSearchPackageListItem = ({ ...props }) => render(
+    <Router history={routerHistory}>
+      <SearchPackageListItem {...props} />
+    </Router>
+  );
+
+  it('should render skeleton when item is not provided', () => {
+    renderSearchPackageListItem(testEmptyProps);
+
+    const isSkeleton = document
+      .querySelector('.skeleton');
+
+    expect(isSkeleton).toBeDefined();
+  });
+
+  it('should render SearchPackageListItem link', () => {
+    renderSearchPackageListItem(testBasicProps);
+
+    const isSearchPackageListItemAttribute = document
+      .querySelector('.item')
+      .getAttribute('data-test-eholdings-package-list-item');
```

Comment:
As @BogdanDenis mentioned, we should use `data-testid` for getting testing elements. Template `data-test-...` was used for testing with `bigtest` library. We don't delete this attributes yet, but we will possibly do it in future with removing `bigtest`